### DISCUSSION TITLE
Will ng-clarity components be available in standalone mode?

### DISCUSSION THREAD

**Discussion by boeckMt** [vmware-clarity/ng-clarity#1585] [General]
It would be nice if the components and related services could be exported so that an application could import them separately without having to read the whole modules.

**Comment by LynxDelli** [comment]
Perhaps you can download the documentation repo.

**Comment by kevinbuhmann** [comment]
There is a module provide for each component.

**Reply by boeckMt** [reply]
I thought it would be nice if a user could take parts of a module and use them to create new components, e.g. just use ClrCalendar and not the whole ClrDatepickerModule.

**Reply by kevinbuhmann** [reply]
We don't have any plans to make the individual pieces of composite components usable on their own. Those non-api-surface components are private implementation details.
